### DISCUSSION TITLE
Deploy multicall3 in Aurora/Aurora testnet

### DISCUSSION THREAD

**Discussion by hskang9** [wevm/wagmi#2172] [Ideas]
Please I am developing a dapp there and I need it now. I will send tokens to deploy.

**Comment by tmm** [comment]
Think you want to open up and issue here: https://github.com/mds1/multicall

> If there is a network Multicall3 is not yet deployed on, please open an issue with a link to the block explorer. You can speed up the new deploy by sending funds to cover the deploy cost to the deployer account: 0x05f32B3cC3888453ff71B01135B34FF8e41263F2

Once deployed, you can add the multicall3 contract address here: https://github.com/wagmi-dev/references
